### PR TITLE
POC/Provisioning:  Support rendering from a nested path 

### PR DESCRIPTION
**Pull request by ryantxu**
The dashboard render from repo currently only works when the file is at the root 😢 

This updates the path so we can see nested files 🎉

### CODE REVIEW

File: public/app/features/dashboard-scene/pages/DashboardScenePage.tsx [hunk 1]
```diff
@@ -21,7 +21,8 @@ export interface Props
 
 export function DashboardScenePage({ route, queryParams, location }: Props) {
   const params = useParams();
-  const { type, slug, uid, path } = params;
+  const { type, slug, uid } = params;
+  const path = params['*'];
```
[the remark this answers is not in the document]
Comment (Clarity-89) [reply public/app/features/dashboard-scene/pages/DashboardScenePage.tsx:25]: This seems to be the best option, apart from doing smth like `pathname.split('/dashboard/')[1]`, but I was also thinking that having paths like this in the URL might not be a good idea. Could we URL encode the path so it becomes e.g., `/admin/provisioning/repositoryp679b/dashboard/panel-flamegraph%2Fpanel_tests_flame_graph.json`, and then we can use `/admin/provisioning/:slug/dashboard/:name` as the route path?

File: public/app/routes/routes.tsx [hunk 2]
```diff
@@ -64,7 +64,7 @@ export function getAppRoutes(): RouteDescriptor[] {
       ),
     },
     {
-      path: '/dashboard/provisioning/:slug/:path/*',
+      path: '/admin/provisioning/:slug/dashboard/*',
```
[the remark this answers is not in the document]
Comment (ryantxu) [reply public/app/routes/routes.tsx:67]: I think it belongs wherever we put the provisioning UI.  So for now admin?

[the remark this answers is not in the document]
Comment (Clarity-89) [reply public/app/routes/routes.tsx:67]: In my thinking, everything related to repo configs, status, etc would be in admin, but the actual page for rendering dashboards would be under dashboard as this seems to be conceptually not related to the provisioning configs. We can keep it under admin tho, if that males more sense to you.

**Comment by Clarity-89** [comment]
Closing in favor of https://github.com/grafana/grafana/pull/97019

**Pull request closed without merging**